packages/web/components/cards/validator-squad-card.tsx

```diff
@@ -1,13 +1,68 @@
+import { Staking } from "@keplr-wallet/stores";
 import React from "react";
 import { useTranslation } from "react-multi-lang";
 
 import { Button } from "~/components/buttons";
 import OsmoverseCard from "~/components/cards/osmoverse-card";
+import { useStore } from "~/stores";
+
+const maxVisibleValidators = 8;
 
 export const ValidatorSquadCard: React.FC<{
   setShowValidatorModal: (val: boolean) => void;
-}> = ({ setShowValidatorModal }) => {
+  validators?: Staking.Validator[];
+  usersValidatorsMap?: Map<string, Staking.Delegation>;
+}> = ({ setShowValidatorModal, validators, usersValidatorsMap }) => {
   const t = useTranslation();
+  const { chainStore, queriesStore } = useStore();
+  const { chainId } = chainStore.osmosis;
+  const queries = queriesStore.get(chainId);
+
+  const queryValidators = queries.cosmos.queryValidators.getQueryStatus(
+    Staking.BondStatus.Bonded
+  );
+
+  let validatorBlock = (
+    <div className="flex flex-row space-x-2">
+      {Array(maxVisibleValidators)
+        .fill(0)
+        .map((_, index) => (
+          <AvatarIcon key={index} />
+        ))}
+    </div>
+  );
+
+  const myValidators = validators?.filter((validator) => {
+    return usersValidatorsMap?.has(validator.operator_address);
+  });
+
+  if (validators?.length && myValidators?.length) {
+    validatorBlock = (
+      <div className="flex flex-row space-x-2">
+        {myValidators?.slice(0, maxVisibleValidators).map((validator) => {
+          const imageUrl = queryValidators.getValidatorThumbnail(
+            validator.operator_address
+          );
+
+          return (
+            <div
+              className="h-10 w-10 overflow-hidden rounded-full"
+              key={validator?.description?.moniker}
+            >
+              <img alt={validator?.description?.moniker} src={imageUrl || ""} />
```

Comment:
I'm not sure we can if we don't know the source? I'll try. Good idea

Reply:
Yea we would need to register each and every domain in our next config. Won't work since each image is from a different domain. I think this is why we keep all token images in our repo, so that we can use Next.